tools/seurat_v5/.shed.yml

```
@@ -0,0 +1,20 @@
+name: seurat_v5
```

```suggestion
name: seurat
```

Lets change it. But I think this will have no effect as it is a tool suite.